lib/geoengineer/resources/aws/api_gateway/aws_api_gateway_resource.rb

```diff
@@ -16,7 +16,11 @@ class GeoEngineer::Resources::AwsApiGatewayResource < GeoEngineer::Resource
 
   # Parent is a resource (the default being the root resource of the rest_api '/')
   # At the moment we only support rest_api being the root resource
```

Comment:
would get rid of this comment since this PR invalidates it

Reply:
Removed :+1: